qutip/solver/integrator/explicit_rk.pxd

```diff
@@ -26,12 +26,12 @@ cdef class Explicit_RungeKutta:
     cdef readonly double rtol, atol, first_step, min_step, max_step
     cdef readonly int max_numsteps
     cdef readonly bint interpolate
-    cdef readonly object method
 
     # Runge Kutta tableau and info
     cdef int rk_step, rk_extra_step, order, denseout_order
-    cdef bint adaptative_step, can_interpolate
+    cdef bint adaptative_step, can_interpolate, fsal
```

Comment:
```suggestion
    cdef bint adaptative_step, can_interpolate, fsal
```
It would be nice to explain `fsal` a little in a comment and give it a better name. I.e. that it means that the last derivative from the previous step is the same as the first derivative from the next step. Since we use longer names for the other values, perhaps we should also rename it `first_derivative_same_as_last` although that seems long.